<launch>

	<node pkg="gmapping" type="slam_gmapping" name="slam_gmapping" output="screen">
		<param name="base_frame" value="base_footprint"/>
		<param name="minimumScore" value="50.0"/>
	    <param name="map_update_interval" value="1.0"/>
	    <param name="maxUrange" value="2.0"/>
	    <param name="maxRange" value="3.8"/>
	    <param name="sigma" value="0.1"/>
	    <param name="kernelSize" value="1"/>
	    <param name="lstep" value="0.15"/>
	    <param name="astep" value="0.15"/>
	    <param name="iterations" value="5"/>
	    <param name="lsigma" value="0.1"/>
	    <param name="ogain" value="3.0"/>
	    <param name="lskip" value="1"/>
	    <param name="srr" value="0.1"/>
	    <param name="srt" value="0.2"/>
	    <param name="str" value="0.1"/>
	    <param name="stt" value="0.2"/>
	    <param name="linearUpdate" value="0.2"/>
	    <param name="angularUpdate" value="0.3"/>
	    <param name="temporalUpdate" value="5"/>
	    <param name="resampleThreshold" value="0.5"/>
	    <param name="particles" value="50"/>
	    <param name="xmin" value="-15.0"/>
	    <param name="ymin" value="-15.0"/>
	    <param name="xmax" value="15.0"/>
	    <param name="ymax" value="15.0"/>
	    <param name="delta" value="0.02"/>
	    <param name="llsamplerange" value="0.01"/>
	    <param name="llsamplestep" value="0.05"/>
	    <param name="lasamplerange" value="0.05"/>
	    <param name="lasamplestep" value="0.05"/>
	    <remap from="/scan" to="/LaserScanner/scan" />
	</node>

	<node pkg="slamtest_package" name="slamtest_node" type="slamtest_node" output="screen">
		<param name="lidar_sub_topic" value="/LaserScanner/scan" />
		<param name="viz_marker_pub_topic" value="/pose_marker" />
		<param name="odom_sub_topic" value="/fmKnowledge/pose" />
		<param name="waypoint_pub_topic" value="/waypoint_goto" />
		<param name="wp_reached_sub_topic" value="/waypoint_reached" />
		<param name="pose_pub_topic" value="/slam_pose" />
		<param name="clicked_point_sub_topic" value="/clicked_point" />
		
	</node>
		
	

</launch>